doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='create-service-task-proc']
 
 = Creating service tasks
-A service task is a task that executes an action based on a web service call or in a Java class method. Examples of service tasks include sending an email and logging a message when these tasks are performed by systems. You can define the parameters (input) and results (output) that are associated with a service task. A Service Task should have one incoming connection and one outgoing connection.
+A service task is a task that executes an action based on a web service call or in a Java class method. Examples of service tasks include sending an email and logging a message when these tasks are performed by systems. You can define the parameters (input) and results (output) that are associated with a service task. You can also define wrapped parameters that contain all input into single objects. A Service Task should have one incoming connection and one outgoing connection.
```

Comment:
all inputs

Reply:
Agreed, Thanks Emily!

Reply:
@csherrar would be interesting to also mention the new parameter that is needed to use this new functionality. 'Wrapped' needs to be set to _true_ .

Reply:
Thanks, @emmurphy1 @cristianonicolai